SOURCES/etc/xapi.d/plugins/updater.py

```diff
@@ -32,30 +32,29 @@ def __init__(self, operation, timeout=0):
         self.operation = operation
 
     def _raise_busy(self):
-        raise OperationException('The updater plugin is busy (current operation: {})'.format(self.current_operation))
-
-    def _prelock(self):
-        # 1. Read the current operation from lockfile before timeout or direct lock call.
+        op = self.current_operation
+        if not op:
+            op = '<UNKNOWN>'
```

Comment:
shouldn't `'<UNKNOWN>'` be stored in a global variable?

Reply:
Why? IMHO It's useful only in this context. Maybe later if we think it's required.

Reply:
Fine by me.